### PR TITLE
enable dev on Ropsten

### PR DESCRIPTION
**Pull request by potuz**
Enable --dev flag when testing on Ropsten.

### CODE REVIEW

**Review by james-prysm — APPROVED** [review]
LGTM

File: config/features/config.go [hunk 1]
```diff
@@ -122,6 +122,7 @@ func configureTestnet(ctx *cli.Context) error {
 		if err := params.SetActive(params.RopstenConfig().Copy()); err != nil {
 			return err
 		}
+		enableDevModeFlags(ctx)
```
Comment (terencechain) [inline config/features/config.go:125]: I'm not sure about enabling all the dev flags. How about just enabling the ones that we need?

Comment (potuz) [reply config/features/config.go:125]: Dev is only two flags afaik

Comment (terencechain) [reply config/features/config.go:125]: The issue is someone could add any flags to dev later that we may not want them on a production testnet. I would prefer this to be flag specific

**Review by terencechain — DISMISSED** [review]

**Review by terencechain — APPROVED** [review]

**Pull request merged**